declarations/Finnair.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Finnair",
+  "documents": {
+    "Privacy Policy": {
+      "fetch": "https://www.finnair.com/fi-en/info/finnair-privacy-policy",
```

```suggestion
      "fetch": "https://www.finnair.com/en/info/finnair-privacy-policy",
```
I think we should track international documents, not English language documents for the Finnish jurisdiction.

---

Changing that is not an issue to change for an already tracked and versioned document? Potentially leading to changes on it?

---

Unless I missed something, it is an addition, so I don't think there is a problem with changing the url now.